src/__tests__/events.js

```diff
@@ -169,9 +172,9 @@ test('onInput works', () => {
 test('calling `fireEvent` directly works too', () => {
```

Comment:
Let's add a test specifically for the issue tagged here

Reply:
I added those to the big list of tests, they should be covered, they were just missing before. With this bug they would have passing tests anyway.

Reply:
The other related issue, I think the bottom issue in open was about events that were already covered in this test suite too. All passed.

Reply:
updated the description to link to the issue as that was missing

Reply:
Would it be worth adding reverse tests? I dont know if if thats a thing but we could write tests that check `'onpointerenter' in dom` and once they fail, we'd know we could delete these patches?

Reply:
I would leave them out for now

Reply:
Worth adding them in another pr? Like I said, looking for ways to contribute